src/stylesheets/pages/_comment.scss

```diff
@@ -0,0 +1,18 @@
+@use '../abstract/' as * ;
+.main-comment{
+    position: relative;
+    width: 1442px;
+    height: 508px;
```

Comment:
check width height. pls!!

Reply:
Pls do not set a fixed width and height if possible. If u need, use max-width and % to set width and min-height to set height. But I do not recommend a set height